src/CMakeLists.txt

```diff
@@ -317,12 +317,13 @@ endif()
 # On clang and gcc, some functionalities of std::atomic require -latomic.
 # See <https://en.cppreference.com/w/cpp/atomic/atomic#Notes>.
 # Note that find_library does not reliably find it so we have to resort to this.
+# Also, passing -latomic is not always the same as adding atomic to the library list.
```

Comment:
Question: What's the difference?

Reply:
I can only assume some code somewhere rewrites `-latomic` because adding `atomic` to the libs failed on MSVC. You can see this in the previous CI runs.